packages/components/addon/components/hds/pagination-bar/pagination/nav-button.hbs

```diff
@@ -0,0 +1,33 @@
+{{! TODO: Add logic to apply class: "hds-pagination__page-link--labeled" }}
+<Hds::Interactive
```

Comment:
@didoo let me know if you think this (and the `pageNumber` child components) should just use html `button` tags instead. (I used `Hds::Interactive` because of previous feedback)